### PR TITLE
[ISSUE #4323] OrderMessage example add the default NamesrvAddr

### PR DESCRIPTION
**Pull request by li-xiao-shuang**
fix : #4323 

**Make sure set the target branch to `develop`**

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Review by Kvicii — DISMISSED** [review]
LGTM

File: example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java [hunk 1]
```diff
@@ -19,35 +19,38 @@
...
+    public static final String TOPIC = "TopicTest";
+    public static final int MESSAGE_COUNT = 100;
+
     public static void main(String[] args) throws UnsupportedEncodingException {
         try {
-            DefaultMQProducer producer = new DefaultMQProducer("please_rename_unique_group_name");
+            DefaultMQProducer producer = new DefaultMQProducer(PRODUCER_GROUP);
+
+            // If the debugging source code can open comments, you need to set the namesrvAddr to the address of the local namesrvAddr
+//            producer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
Comment (Oliverwqcwrw) [inline example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java:41]:
```suggestion
          //  producer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
My obsessive compulsive disorder

File: example/src/main/java/org/apache/rocketmq/example/ordermessage/Consumer.java [hunk 2]
```diff
@@ -28,12 +28,19 @@
...
+    public static final String DEFAULT_NAMESRVADDR = "127.0.0.1:9876";
+    public static final String TOPIC = "TopicTest";
+    public static final String SUB_EXPRESSION = "TagA || TagC || TagD";
+
     public static void main(String[] args) throws MQClientException {
-        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("please_rename_unique_group_name_3");
+        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(CONSUMER_GROUP);
 
+        // If the debugging source code can open comments, you need to set the namesrvAddr to the address of the local namesrvAddr
+//        consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
Comment (Oliverwqcwrw) [inline example/src/main/java/org/apache/rocketmq/example/ordermessage/Consumer.java:40]:
```suggestion
      //  consumer.setNamesrvAddr(DEFAULT_NAMESRVADDR);
```
My obsessive compulsive disorder

**Review by Oliverwqcwrw — CHANGES_REQUESTED** [review]

File: example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java (hunk 1 above)
Comment (li-xiao-shuang) [reply example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java:41]: This is the normal format

**Review by hzh0425 — APPROVED** [review]

**Pull request closed without merging**